modules/rest-api/pages/rest-identify-orchestrator.adoc

```diff
@@ -0,0 +1,21 @@
+= Identifying the Orchestrator Node
+
+[abstract]
+The node currently running the _orchestrator_ (sometimes referred to as the _master services_) can be identified by means of the REST API
+
+== HTTP Method and URI
+
+----
+GET /pools/default/terseClusterInfo
+----
+
+== Description
+
+The _orchestrator_ manages cluster-wide operations; such as vBucket-placement, failover, node addition and subtraction, rebalance, cluster configuration-mapping, and aggregation of statistical data.
```

Comment:
"node addition" and "aggregation of statistical data", are not in the orchestrator's job description.  Also, not sure what is meant by "cluster configuration-mapping".

I would suggest wording as below,
"The _orchestrator_ handles operations with cluster-wide impact; such as failover, rebalance,  adding/deleting buckets, etc. "

Reply:
Thanks, have modified accordingly.